.changeset/shaggy-readers-argue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@easypost/easy-ui": minor
```

Comment:
**Question: ** Do we add changes for new icon files?

Reply:
ah yes, youre right, updated PR